src/components/Cards.jsx

```diff
@@ -0,0 +1,87 @@
+import { React, useState, useEffect } from "react";
+import BackgroundUrls from "./BackgroundUrls";
+
+const Cards = () => {
+  const [loading, setLoading] = useState(true);
+  const [data, setData] = useState(null);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    fetch(`https://swapi.dev/api/films`)
+      .then((response) => {
+        if (!response.ok) {
+          throw new Error(`Status is ${response.status}`);
+        }
+        return response.json();
+      })
+      .then((rawData) => {
+        console.log(rawData);
+        setData(rawData.results);
+        setError(null);
+      })
+      .catch((error) => {
+        console.log(error);
+        setError(error);
+        setData(null);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
+  }, []);
```

Comment:
Good use of useEffect to handle the data fetching side-effect

Reply:
Thank you